fiftyone/core/odm/database.py

```diff
@@ -128,39 +134,25 @@ def get_db_config():
     return config
 
 
-def cleanup_multiple_config_docs():
+def cleanup_multiple_config_docs(conn, config_docs):
     """Internal utility that ensures that there is only one
     :class:`DatabaseConfigDocument` in the database.
     """
 
-    # We use mongoengine here because `get_db_conn()` will not currently work
-    # until `import fiftyone` succeeds, which requires a single config doc
-    # pylint: disable=no-member
-    docs = list(DatabaseConfigDocument.objects)
-    if len(docs) <= 1:
-        return
+    if not config_docs:
+        return {}
+    elif len(config_docs) <= 1:
+        return config_docs[0]
 
     logger.warning(
         "Unexpectedly found %d documents in the 'config' collection; assuming "
         "the one with latest 'version' is the correct one",
-        len(docs),
+        len(config_docs),
     )
+    keep_doc = max(config_docs, key=lambda d: Version(d.get("version", 0)))
```

Comment:
This seems dangerous to potentially revert to a version `0`. It's never happened to my knowledge, but currently if no valid config document is found, a hard exception occurs

Reply:
Hmm well if there's a hard error then that would prevent migration for the version to be fixed. So you'd be stuck.

I will test it, I think it should be treated the same as no config doc (v0.14.4)

Reply:
If there's no config document it assumes v0.14.4 but doesn't hard error.

Reply:
Right, but if there is >=1 docs, all without a version value, the behavior has changed. But it's and odd situation

Reply:
Just tested on `develop` and behavior has not changed, what I said above is true for this case.
However I will push a commit because an integer is no bueno for `Version`, so we will use `"0.0"`.

Reply:
fixed/pushed